gui-wm/dwl/dwl-9999-r1.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit savedconfig toolchain-funcs
+
+WLROOTS_SLOT="0/16"
+if [[ ${PV} == *9999* ]]; then
+	EGIT_REPO_URI="https://github.com/djpohly/dwl"
+	inherit git-r3
+
+	# 9999-r0: main (latest stable wlroots release)
+	# 9999-r1: wlroots-next (wlroots-9999)
+	case ${PVR} in
+		9999)
+			EGIT_BRANCH=main
+			;;
+		9999-r1)
+			EGIT_BRANCH=wlroots-next
+			WLROOTS_SLOT="0/9999"
+			;;
+	esac
+else
+	SRC_URI="https://github.com/djpohly/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="dwm for Wayland"
+HOMEPAGE="https://github.com/djpohly/dwl"
+
+LICENSE="CC0-1.0 GPL-3+ MIT"
+SLOT="0"
+IUSE="X"
+
+RDEPEND="
+	dev-libs/libinput:=
+	dev-libs/wayland
+	gui-libs/wlroots:${WLROOTS_SLOT}[X(-)?]
+	x11-libs/libxkbcommon
+	X? (
+		x11-libs/libxcb:=
+		x11-libs/xcb-util-wm
+	)
+"
+
+# uses <linux/input-event-codes.h>
+DEPEND="
+	${RDEPEND}
+	sys-kernel/linux-headers
+"
+BDEPEND="
+	>=dev-libs/wayland-protocols-1.32
+	dev-util/wayland-scanner
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	restore_config config.h
+
+	default
+}
+
+src_compile() {
+	emake PKG_CONFIG="$(tc-getPKG_CONFIG)" CC="$(tc-getCC)" \
+		XWAYLAND="$(usex X -DXWAYLAND "")" XLIBS="$(usex X "xcb xcb-icccm" "")" dwl
```

Comment:
please use usev

Reply:
How do I use it?

manpage says:
```
       usev <USE item>
              Like use, but also echoes USE item when use returns true.
```

Reply:
Ah, nevermind I found it: https://mgorny.pl/articles/the-ultimate-guide-to-eapi-8.html#usev-now-accepts-a-second-argument

The manpage should be updated.

Reply:
Would you mind filing a bug for the man page too? I filed https://bugs.gentoo.org/914590 earlier for the devmanual.

Reply:
[Done](https://bugs.gentoo.org/914613)